### PR TITLE
[New Rule] AWS ElastiCache Security Group Modified or Deleted

### LINKED ISSUE #1352: [New Rule] AWS ElastiCache Security Group Modified or Deleted
## Description
Identifies when an ElastiCache security group has been modified.

### Target indexes

`filebeat-*, logs-aws*`

### Platforms

AWS Cloudtrail

### Query

```
event.dataset:aws.cloudtrail and  event.provider:elasticache.amazonaws.com and 
event.action:(Delete Cache Security Group or Authorize Cache Security Group Ingress or 
Revoke Cache Security Group Ingress or AuthorizeCacheSecurityGroupEgress or RevokeCacheSecurityGroupEgress) and 
event.outcome:success
```

### **False Positives**
- 

### MITRE

**ATTACK TACTIC**
Credential Access, Persistence

**ATTACK TECHNIQUE**
Account Manipulation

### References
- https://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/Welcome.html

### PR DESCRIPTION
**Pull request by austinsonger**
## Issues

Resolves #1352

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/integrations/aws/impact_elasticache_security_group_modified_or_deleted.toml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+[[rule.threat.technique]]
+id = "T1531"
+name = "Account Access Removal"
+reference = "https://attack.mitre.org/techniques/T1531/"
+
+
+[rule.threat.tactic]
+id = "TA0040"
+name = "Impact"
+reference = "https://attack.mitre.org/tactics/TA0040/"
```
Comment (w0rk3r) [inline rules/integrations/aws/impact_elasticache_security_group_modified_or_deleted.toml:52]:
```suggestion
[[rule.threat]]
framework = "MITRE ATT&CK"
[[rule.threat.technique]]
id = "T1562"
name = "Impair Defenses"
reference = "https://attack.mitre.org/techniques/T1562/"

  [[rule.threat.technique.subtechnique]]
  id = "T1562.007"
  name = "Disable or Modify Cloud Firewall"
  reference = "https://attack.mitre.org/techniques/T1562/007/"

[rule.threat.tactic]
name = "Defense Evasion"
id = "TA0005"
reference = "https://attack.mitre.org/tactics/TA0005/"
```

This one fits better in this one, we also need to rename the rule to `defense_evasion_elasticache_security_group_modified_or_deleted.toml`

**Review by w0rk3r — CHANGES_REQUESTED** [review]
Just one requested change, then LGTM.

File: rules/integrations/aws/impact_elasticache_security_group_modified_or_deleted.toml (hunk 1 above)
Comment (w0rk3r) [reply rules/integrations/aws/impact_elasticache_security_group_modified_or_deleted.toml]: The file needs to be renamed to the above to correspond with the tactic.

**Review by w0rk3r — APPROVED** [review]
LGTM

**Review by brokensound77 — APPROVED** [review]
👍

**Pull request merged**